test/libsolidity/smtCheckerTests/overflow/signed_div_overflow.sol

```diff
@@ -6,6 +6,5 @@ contract C  {
 	}
 }
 // ----
-// Warning 1218: (110-115): CHC: Error trying to invoke SMT solver.
+// Warning 4984: (110-115): CHC: Overflow (resulting value larger than 0x80 * 2**248 - 1) happens here.
```

Comment:
It seems like the CHC engine should also detect the possible division of zero. What do you think, @leonardoalt ?

Reply:
@blishko yep! It currently doesn't check for it, but we should add a check.

Reply:
You mean the check is not part of the encoding now? How should the check be added?

Reply:
I mean that BMC creates a verification target for division by 0, but CHC doesn't. It can be added in `CHC::arithmeticOperation` I think, similar to the underflow/overflow verification targets.

Reply:
But we can do that in a different PR